webcompat/helpers.py

```diff
@@ -447,7 +447,7 @@ def proxy_request(method, path, params=None, headers=None, data=None):
     return req(resource_uri, data=data, params=params, headers=auth_headers)
 
 
-def api_request(method, path, params=None, data=None):
+def api_request(method, path, params=None, data=None, mime_type=JSON_MIME):
```

Comment:
where is `JSON_MIME` defined now?

Reply:
JSON_MIME is defined at the top of this file, as it was before. 
When it's necessary I pass `JSON_MIME_HTML`as an argument to `mime_type`, like I do [here](https://github.com/webcompat/webcompat.com/pull/2731/files#diff-b9ca02974fd8e51b2c21afc1f20d62b5). Do you think I should write a comment on the code, explaining why do we use two different kind of mimes?

Reply:
ah I missed it. Apologizes. thanks. No need for comments at this point.